bin/jscs

```diff
@@ -15,6 +15,7 @@ program
     .usage('[options] <file ...>')
     .description('A code style linter for programmatically enforcing your style guide.')
     .option('-c, --config [path]', 'configuration file path')
+    .option('--configure [path]', 'auto-generate a JSCS configuration file')
```

Comment:
`--configure` and `--config`, `-c` are very similar. Can we try another name for it?